en/07_Depth_buffering.adoc

```diff
@@ -293,23 +299,23 @@ We don't need to map it or copy another image to it, because we're going to clea
 === Explicitly transitioning the depth image
 
 We don't need to explicitly transition the layout of the image to a depth attachment because we'll take care of this in the render pass.
-However, for completeness I'll still describe the process in this section.
+However, for completeness, I'll still describe the process in this section.
 You may skip it if you like.
 
 Make a call to `transitionImageLayout` at the end of the `createDepthResources` function like so:
 
 [,c++]
 ----
-transitionImageLayout(depthImage, depthFormat, VK_IMAGE_LAYOUT_UNDEFINED, VK_IMAGE_LAYOUT_DEPTH_STENCIL_ATTACHMENT_OPTIMAL);
+transitionImageLayout(depthImage, depthFormat, vk::ImageLayout::eUndefined, vk::ImageLayout::eTransferDstOptimal);
 ----
 
 The undefined layout can be used as initial layout, because there are no existing depth image contents that matter.
-We need to update some of the logic in `transitionImageLayout` to use the right subresource aspect:
+We need to update some logic in `transitionImageLayout` to use the right subresource aspect:
 
 [,c++]
 ----
-if (newLayout == VK_IMAGE_LAYOUT_DEPTH_STENCIL_ATTACHMENT_OPTIMAL) {
-    barrier.subresourceRange.aspectMask = VK_IMAGE_ASPECT_DEPTH_BIT;
+if (newLayout == vk::ImageLayout::eDepthStencilAttachmentOptimal) {
+    barrier.subresourceRange.aspectMask = vk::ImageAspectFlagBits::eDepth;
```

Comment:
The block startig at line 334 (which I can't tag here) need to be updated, it still uses the C header style.